src/SettingsPage/View.php

```diff
@@ -54,11 +66,62 @@ public function add_options_page() {
 		);
 	}
 
+	/**
+	 * Enqueue scripts and styles necessary for the Settings Page.
+	 * 
+	 * Only executed when the Settings Page is actually displayed.
+	 *
+	 * @return void
+	 */
+	public function enqueue_scripts_styles() {		
+		$screen = get_current_screen();
+
+		if ( $screen->id === 'settings_page_adminimize' ) {
+			wp_enqueue_script(
+				'adminimize_admin',
+				plugins_url( '../../assets/js/adminimize.js', __FILE__ ),
+				[ 'jquery', 'jquery-ui-tabs' ]
+			);
+		}		
+	}
+
 	/**
 	 * HTML and Content for the settings page.
 	 */
 	public function render_page() {
+		$this->get_tabs();
 
 		include $this->settings_page->get_template_path() . '/Templates/SettingsPage.php';
 	}
+
+	/**
+	 * Get instances of all tabs.
+	 *
+	 * @return void
+	 */
+	private function get_tabs() {
+		$tabs = new Tabs\Tabs;
+
+		$tabs_list = $tabs->get_tabs_list();
+
+		foreach ( $tabs_list as $tab_class ) {
+			if ( ! is_null( $tab = $this->instantiate_tab( $tab_class ) ) ) {
+				$this->tabs[] = $tab;
+			}
+		}
+	}
+
+	/**
+	 * Instantiate single tab if class exists.
+	 *
+	 * @param string $tab_class Class name of Tab to be instantiated
+	 * @return Tabs\TabInterface|null
+	 */
+	private function instantiate_tab( string $tab_class ) : ?Tabs\TabInterface {
+		if ( class_exists( $tab_class ) ) {
+			return new $tab_class( $this->settings_page );
+		} else {
```

Comment:
I think the else is not necessary. The if check run in a return, if `if` will fail, the default return is sufficient.